### ISSUE TITLE
Check for entry == duration?

### ISSUE THREAD

**Issue by jseabold** [CamDavidsonPilon/lifelines#1166]
I was getting a NaN during the gradient calculations while fitting for a CoxPH with late entry, and I think I tracked it down to having someone who had an event on the same day the observation period started, which meant for one time that [this was all False](https://github.com/CamDavidsonPilon/lifelines/blob/v0.25.6/lifelines/fitters/cox_time_varying_fitter.py#L530). I made entry the day before the observations started and all seems well now. Does it make sense to change the [entry check](https://github.com/CamDavidsonPilon/lifelines/blob/v0.25.6/lifelines/utils/__init__.py#L1122) from > to >=?

**Comment by CamDavidsonPilon** [comment]
👋 hi @jseabold,

Hm, I would have to think more about _why_ the convergence fails when entry == duration. I would have thought it would be okay, but evidently not. 

Anyways, probably annoying that you had to track this down, but I appreciate you finding the problem and reporting it to me.

**Comment by jseabold** [comment]
Howdy!

> probably annoying that you had to track this down

Not a problem. The docs and code have been really helpful to go through.

> why the convergence fails when entry == duration

Yeah, I didn't fully figure it out, obviously. In this case there were two people one censored and one had event at t. Once I got that far, I just stared at the data and figured that was what was happening. I would have expected one of the deaths to be 1 though. I'll have to stare at it some more too.
